book/chapters/chapter3/evaluation_and_benchmarking.qmd

```diff
@@ -248,46 +248,38 @@ While we are usually interested in this aggregated score, it can be useful to lo
 include_multi_graphics("mlr3book_figures-5")
 ```
 
-To visualize the resampling results, you can use the `r ref("mlr3viz::autoplot.ResampleResult()")` function to plot scores across folds as boxplots (@fig-resamp-viz-1) or histograms (@fig-resamp-viz-2).
+To visualize the resampling results, you can use the `r ref("mlr3viz::autoplot.ResampleResult()")` function to plot scores across folds as boxplots or histograms (@fig-resamp-viz).
 Histograms can be useful to visually gauge the variance of the performance results across resampling iterations, whereas boxplots are often used when multiple learners are compared side-by-side (see @sec-benchmarking).
 
-```{r performance-035}
-#| output: false
-#| cache: false
-rr = resample(tsk_penguins, lrn_rpart, rsmp("cv", folds = 10))
-autoplot(rr, measure = msr("classif.acc"), type = "boxplot")
-```
 
-```{r}
-#| label: fig-resamp-viz-1
-#| fig-cap: "Boxplot of accuracy scores."
-#| fig-alt: "A boxplot ranging from 0.875 to 1.0 and the interquartile range between 0.925 and 0.7."
-#| echo: false
-#| warning: false
-#| message: false
-plt = ggplot2::last_plot()
-plt$layers[[1]]$aes_params$fill = "white"
-print(plt)
-```
 
 ```{r}
-#| output: false
-#| cache: false
+#| eval: false
```

Comment:
Should this really be eval: false ?

Reply:
yes, (all the code is run in the previous chunk)